packages/tocco-ui/src/EditableValue/typeEditors/MultiDateEdit.js

```diff
@@ -0,0 +1,165 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import moment from 'moment'
+import {injectIntl, intlShape} from 'react-intl'
+
+import DateAbstract from './DateAbstract'
+import {atMostOne, momentJStoToFlatpickrFormat, toLocalDateString} from '../utils'
+
+class MultiDateEdit extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      showCalendar: true,
+      showDoubleCalendar: false,
```

Comment:
If I understand the code correctly, `showCalendar` and `showDoubleCalendar` are connected. If `showCalendar` is `false`, `showDoubleCalendar` **must** be `true` and vice versa.

Hence, in my option, it would make sense to reduce it to one property. It could be called `view` or `mode` or something like that and accept the values `exact` or `range`.